Lib/idlelib/replace.py

```diff
@@ -120,7 +120,7 @@ def _replace_expand(self, m, repl):
         if self.engine.isre():
             try:
                 new = m.expand(repl)
-            except re.error:
+            except re.PatternError:
```

Comment:
This was removed in the original PR.

Reply:
This change was not in the original diff for this issue.  I definite want both or neither and I like the change enough to do both, even though they cannot be backported and there is a faint chance that this could block a future auto-backport.  In other words, approved as is now.  I am glad you caught this.  I checked and there are only these 2 occurrences of re.error and none of `except error`.